src/table/TableHeadWrapper.jsx

```diff
@@ -64,7 +64,7 @@ export default function TableHeadWrapper({
                 }
                 tabIndex={-1}
               >
-                {column.label}
+                <div>{column.label}</div>
```

Comment:
Hmm not sure about this. How does it affect the screen reader?

Reply:
<img width="677" alt="Screenshot 2021-05-04 at 10 21 55" src="https://user-images.githubusercontent.com/4471489/116977360-a235cd80-acc2-11eb-9fcc-ecb0320acb65.png">
looks good in sense-client

Reply:
<img width="801" alt="Screenshot 2021-05-04 at 10 25 52" src="https://user-images.githubusercontent.com/4471489/116977778-1ff9d900-acc3-11eb-8076-1c6fa11eedf6.png">
looks good in sense-client